applications/GettingStartedWith/DateRangeBox/index.js

```diff
@@ -0,0 +1,15 @@
+$(function() {
+    $("#dateRangeBox").dxDateRangeBox({
+        startDate: new Date(),
+        endDate: new Date(new Date().setDate(new Date().getDate() + 7)),
+        min: new Date(new Date().setDate(new Date().getDate() - 14)),
+        max: new Date(new Date().setDate(new Date().getDate() + 14)),
+        startDateLabel: "Start",
+        endDateLabel: "End",
+        labelMode: "floating",
+        displayFormat: "EEEE, d 'of' MMM", // "Tuesday, 19 of Oct - Sunday, 24 of Oct" 
```

Comment:
I discussed this date format with Vladimir, and we both agree that this format looks strange. Can you change it to the following format instead?

`displayFormat: "EEEE, MMM d", // "Tuesday, Oct 19 - Sunday, Oct 24"`

Reply:
Yes